blinkpy/auth.py

```diff
@@ -74,9 +82,9 @@ def header(self):
         if self.token is None:
             return None
         return {
-            "APP-BUILD": self._app_build,
-            "TOKEN_AUTH": self.token,
-            "User-Agent": self._agent,
+            # "APP-BUILD": self._app_build,
```

Comment:
@fronzbot I commented APP BUILD and User-Agent because with the current value we have APIs don't work. Without them it's working everything.

Idk if we are able to get new values or just do not use them